kubernetes_state_core/README.md

```diff
@@ -0,0 +1,59 @@
+# Agent Check: Kubernetes State Core
+
+## Overview
+
+Get Kubernetes metrics from [`kube-state-metrics`](https://github.com/kubernetes/kube-state-metrics) in real time to:
```

Comment:
Could you add this link at the bottom of the file with the other links please? Thanks!